Lombiq.Tests.UI/Pages/OrchardCoreSetupPage.cs

```diff
@@ -68,20 +70,22 @@ public enum DatabaseType
 
         public Button<_> FinishSetup { get; private set; }
 
-        public _ ShouldStayOnSetupPage() =>
-            PageTitle.Should.Equal(DefaultPageTitle);
+        public _ ShouldStayOnSetupPage() => PageTitle.Should.Satisfy(title => IsExpectedTitle(title));
 
-        public _ ShouldLeaveSetupPage() =>
-            PageTitle.Should.Not.Equal(DefaultPageTitle);
+        public _ ShouldLeaveSetupPage() => PageTitle.Should.Not.Satisfy(title => IsExpectedTitle(title));
 
-        [Obsolete("Use another overloaded " + nameof(SetupOrchardCore) + " method without UITestContext parameter.")]
+        [Obsolete("Use another overload of this method without a " + nameof(UITestContext) + " parameter.")]
         public _ SetupOrchardCore(UITestContext context, OrchardCoreSetupParameters parameters = null) =>
             SetupOrchardCore(parameters);
 
         public _ SetupOrchardCore(OrchardCoreSetupParameters parameters = null)
         {
             parameters ??= new OrchardCoreSetupParameters();
 
+            // Set the setup page window size to avoid the "OpenQA.Selenium.WebDriverException: move target out of
+            // bounds" that may happen in headless mode due to low default window size.
+            Driver.Manage().Window.Size = CommonDisplayResolutions.Fhd;
```

Comment:
The consumer might has set this already, like `Lombiq.Tests.UI.Samples.Helpers.SetupHelpers`. We shouldn't always override it.

Reply:
It's not just for the setup page but also for the operations coming after it. For the setup page normally I wouldn't think so but keep in mind that you can override its styling (or completely) so people can have a custom setup.

Reply:
Do you mean there may be operations after setup inside the setup window?

Reply:
In the same browser window, yes.

Reply:
Wouldn't it be better to have a config property for the window size?

Reply:
We could have it as a parameter. That overload was obsoleted because we didn't need the context.

I wouldn't really put that into a config since we have statically typed ways of setting the resolution.